build.gradle

```diff
@@ -34,6 +34,7 @@ allprojects {
         cordaCliBinDir = "${System.getProperty("user.home")}/.corda/cli"
         cpiUploadTimeout = cpiUploadDefault
         cordaProcessorTimeout = processorTimeout
+        skipTestsDuringBuildCpis = this.skipTestsDuringBuildCpis

```

Comment:
Would it be better to set this to true? Seems more convenient for a user to opt-in rather than have it set as false and their build time be longer

Reply:
I was going for "slow and safe" by default, and then those who understand what is going on can opt to skip the tests. Happy to discuss, and we can change it at anytime

Reply:
Gotcha 👍